test/test-cases/regression/rule-920200.json

```diff
@@ -18,7 +18,7 @@
         "Accept-Language":"en-us,en;q=0.5",
         "Accept":"*/*",
         "Keep-Alive":"300",
-	"Range": "bytes=1-10,11-20,21-30,31-40,41-50,51-60"
+        "Range": "bytes=1-10,11-20,21-30,31-40,41-50,51-60"
```

Comment:
space change?

Reply:
Yes, an identation fix. Unfortunately it does not seem here in this diff output.